.github/workflows/push-pr-unit-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: unit tests
+
+on: [push, pull_request, workflow_dispatch]
+
+jobs:
+  push-pr-unit-tests:
+    name: unit tests
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+    uses: ./.github/workflows/reusable-unit-tests.yml
+    with:
+      python-version: ${{ matrix.python-version }}
+      branch:
+  push-pr-unit-tests-docker:
+    name: Docker Unit Tests
+    uses: ./.github/workflows/reusable-unit-tests-docker.yml
+    with:
+      branch:
```

Comment:
This looks weird, is this required?

Reply:
Passing an empty branch makes `actions/checkout@v3` use the default one. That's what we want here. If we mark the input as `required: false`, we can probably drop this. I'd have to look what is passed in that case.

Reply:
Fixed.